core-services/prow/02_config/_plugins.yaml

```diff
@@ -173,13 +173,7 @@ bugzilla:
       target_release: 4.3.z
       validate_by_default: true
     openshift-4.4:
-      dependent_bug_states:
-      - status: MODIFIED
-      - status: ON_QA
-      - status: VERIFIED
-      - status: RELEASE_PENDING
-      - resolution: ERRATA
-        status: CLOSED
+      dependent_bug_target_release: 4.5.0
```

Comment:
shouldn't this have `validate_by_default`

Reply:
Yes

Reply:
It does, two lines down :)